.github/workflows/resolve_merge_conflict.yml

```
@@ -0,0 +1,16 @@
+name: 'Resolve Merge Conflicts'
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  resolve-merge-conflicts:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```

I'd recommend pinning the version of Ubuntu, so that things don't suddenly start failing. In https://github.com/web-platform-tests/wpt-metadata/pull/1155 I pinned everything to ubuntu-20.04, so suggesting the same here.